src/pages/experiments/[experimentId]/data-processing/data-integration/components/CalculationConfig.jsx

```diff
@@ -1,43 +1,175 @@
 import React from 'react';
 
 import {
-  Space, Select, InputNumber, Form, Checkbox,
+  Space,
+  Select,
+  InputNumber,
+  Form,
+  Checkbox,
+  Tooltip,
+  Button,
+  Typography,
+  Alert,
+  Row,
+  Col,
 } from 'antd';
 
+import {
+  InfoCircleOutlined,
+} from '@ant-design/icons';
+
 const { Option } = Select;
+const { Text } = Typography;
+
+const CalculationConfig = () => {
+  const methodOptions = [
+    {
+      value: 'seuratV4',
+      text: 'Seurat v4',
+      disabled: false,
+    },
+    {
+      value: 'seuratV3',
+      text: 'Seurat v3',
+      disabled: true,
+    },
+    {
+      value: 'harmony',
+      text: 'Harmony',
+      disabled: true,
+    },
+    {
+      value: 'conos',
+      text: 'Conos',
+      disabled: true,
+    },
+    {
+      value: 'liger',
+      text: 'Liger',
+      disabled: true,
+    },
+    {
+      value: 'fastMNN',
+      text: 'Fast MNN',
+      disabled: true,
+    },
+  ];
+
+  const changesOutstanding = false;
+
+  return (
+    <>
+      <Space direction='vertical' style={{ width: '100%' }} />
+      <Form size='small'>
+        <Form.Item>
+          <Text>
+            <span style={{ marginRight: '0.5rem' }}>Set the parameters for Data Integration</span>
+            <Tooltip title='Integration of multiple samples corrects for batch effect. These methods identify shared cell states that are present across different datasets, even if they were collected from different individuals, experimental conditions, technologies, or even species. The user selects the integration method and sets the controls, as appropriate. The latest Seurat method is selected as default.'>
+              <Button icon={<InfoCircleOutlined />} />
+            </Tooltip>
+          </Text>
+        </Form.Item>
+        <div style={{ paddingLeft: '1rem' }}>
+          <Form.Item
+            label='Method:'
+          >
+            <Select
+              defaultValue='seuratV4'
+            >
+              {
+                methodOptions.map((el) => (
+                  <Option value={el.value} disabled={el.disabled}>{el.text}</Option>
+                ))
+              }
+            </Select>
+          </Form.Item>
+          <Form.Item label='# of genes:'>
+            <InputNumber
+              defaultValue={2000}
+              min={1}
+              onPressEnter={() => { }}
+              onChange={() => { }}
```

Comment:
Do we need to have these events that do nothing there before functionality is added?

Reply:
Those are templates for future work, so that the person doing the next ticket to implement the functionality can continue from there. Maybe I should put `// TODO` or something to signify that.

Reply:
Maybe @ivababukova or @marcellp has something to say about this practice? Is it good to leave templates / todo behind for future works? Or just keep the codebase clean for the PR?

Reply:
It depends. It is usually best to leave the code clean and without todos, commented out code is not a best practice either. If the todo is big enough, make a ticket. If the todo is small enough, implement it. The reason is that even if you put a todo in the code, it is very likely that it won't get done, but just stay there, which is not great. 

In this case however, since you are saying that this will get implemented in the next ticket, there is little chance that the code will be forgotten and left like this. In this case either way is fine -- having the empty functions or not.

Reply:
Ok... I think it's better to erase the hanging functions.